### PR TITLE
Prepare for 4.0.0 release

### PR DESCRIPTION
**Pull request by jsmartt**
### Description
Prepare for 4.0.0 release

### Issues Resolved
(none)

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api200/logical_interconnect_group.rb [hunk 1]
```diff
@@ -71,15 +71,6 @@ def add_uplink_set(uplink_set)
         @data['uplinkSets'] << uplink_set.data
       end
 
-      # Get the logical interconnect group default settings
-      # @deprecated Use {::get_default_settings} instead. This method will be removed in the next major release.
```
Comment (jsmartt) [inline lib/oneview-sdk/resource/api200/logical_interconnect_group.rb]: We said we'd remove this deprecated method in the next major release, and here it is, so I'm removing it

File: lib/oneview-sdk/cli.rb [hunk 2]
```diff
@@ -114,7 +114,7 @@ def version
     # Show environment variables for oneview-sdk-ruby
     def env
       data = {}
-      OneviewSDK::ENV_VARS.each { |k| data[k] = ENV[k] unless k.include?('I3S') }
+      OneviewSDK::ENV_VARS.each { |k| data[k] = ENV[k] }
```
Comment (jsmartt) [inline lib/oneview-sdk/cli.rb]: I also updated the CLI to output the I3S environment variables. The CLI doesn't necessarily use them yet, but the I3S client does

**Comment by jsmartt** [comment]
Other than that it's all documentation updates. Please let me know if I missed anything @tmiotto, @aalexmonteiro, @ricardogpsf, and @fgbulsoni ! I'm ready to :shipit:

**Comment by aalexmonteiro** [comment]
@jsmartt , Looks good for me. :)

**Comment by ricardogpsf** [comment]
I did not see some wrong

**Comment by jsmartt** [comment]
Thanks guys. @tmiotto or @fgbulsoni , I'll wait for the thumbs up from one of you.

**Pull request merged**